Assignment9/routes/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  # get '/users' , to: 'users#index'
```

Comment:
You can remove commented code if its not useful

Reply:
done